### PR TITLE
disallow concurrent builds

### PR DESCRIPTION
**Pull request by cosunae**
Description
-------------------
the variable $WORKSPACE is assigned dynamically at the beginning of every stage
and might change depending on the number of concurrent builds active.
We can only allow 1 concurrent build to have a consistent access to $WORKSPACE
Otherwise we should use stash/unstash for the miniconda installation

### CODE REVIEW

**Comment by cosunae** [comment]
launch jenkins

File: Jenkinsfile [hunk 1]
```diff
@@ -2,6 +2,13 @@ def BuildBadge = addEmbeddableBadgeConfiguration(id: "build", subject: "Build")
 def TestBadge = addEmbeddableBadgeConfiguration(id: "test", subject: "Test")
 
 pipeline {
+    options {
+    # the variable $WORKSPACE is assigned dynamically at the beginning of every stage
```
Comment (victoria-cherkas) [inline Jenkinsfile:6]: comments should be with // instead of # I think? Failing on Jenkins because of this

**Comment by cosunae** [comment]
launch jenkins

File: Jenkinsfile (hunk 1 above)
Comment (cosunae) [reply Jenkinsfile]: yes indeed, thanks

**Review by victoria-cherkas — APPROVED** [review]

**Pull request merged**